plugins/Goals/lang/en.json

```
@@ -68,30 +68,30 @@
         "Filename": "filename",
         "GoalConversion": "Goal conversion",
         "GoalConversions": "Goal conversions",
-        "GoalConversionsBy": "Goal %s conversions by type of visit",
-        "GoalIsTriggered": "Goal is triggered",
-        "GoalIsTriggeredWhen": "Goal is triggered when",
+        "GoalConversionsBy": "%s goal conversions by type of visit",
+        "GoalIsTriggered": "A goal is triggered",
+        "GoalIsTriggeredWhen": "A goal is triggered when",
         "GoalName": "Goal Name",
         "Goals": "Goals",
         "NGoals": "%s goals",
         "NRevenue": "%s revenue",
         "NItems": "%s items",
         "ManageGoals": "Manage Goals",
-        "GoalsOverview": "Goals Overview",
-        "GoalsOverviewDocumentation": "This is an overview of your goal conversions. Initially, the graph shows the sum of all conversions. %s Below the graph, you can see conversion reports for each of your goals. The sparklines can be enlarged by clicking on them.",
-        "GoalX": "Goal %s",
-        "HelpOneConversionPerVisit": "If a Page matching this Goal is refreshed or viewed more than once in a Visit, the Goal will only be tracked the first time the page was loaded during this visit.",
+        "GoalsOverview": "Overview of Goals",
+        "GoalsOverviewDocumentation": "An overview of your goal conversions. Initially, the graph shows the sum of all conversions. %s Below the graph, you can see conversion reports for each of your goals. The sparklines can be enlarged by clicking on them.",
+        "GoalX": "%s Goal",
```

The placeholder will also be replaced with the name of the goal
```suggestion
        "GoalX": "Goal %s",
```

---

Maybe we should use quotes here then as well.